### ISSUE TITLE
Please indicate that the Azure Storage Queue Stream consumer is to delete consumed messages from the Azure Queue

### ISSUE THREAD

**Issue by shawndewet** [akkadotnet/Alpakka#1141]
**Please describe what you are trying to understand**
In trying to build my first Actor that sets up a Stream over Azure Storage Queue, I was stuck for a few hours trying to figure out why the Stream keeps returning the same message over and over.

**Which pages have you looked at?**
https://alpakka.getakka.net/documentation/Azure/StorageQueue.html

**What did these pages not make clear?**
That the Stream provider does not delete delivered messages from the Azure Storage Queue.

**How can we do it better?**
The page should make it clear that deletion of consumed messages from the Azure Storage Queue is the responsibility of the message consumer.

**Comment by Aaronontheweb** [comment]
@shawndewet I agree - we need to make some things clearer in our documentation for Alpakka.

**Comment by shawndewet** [comment]
I finally realized this when I read the Microsoft documentation on Azure Storage Queues [here](https://docs.microsoft.com/en-us/azure/storage/queues/storage-tutorial-queues?tabs=dotnet%2Cenvironment-variable-windows#dequeue-messages).
